### PR TITLE
add note about the de440s case

### PR DESCRIPTION
**Pull request by eteq**
### Description

This is a docs-only PR that basically just draws attention to some of the things that were revealed in astropy/astropy#11608 about some oddities with the latest JPL ephemerides.  We will hopefully work out more detail on this for a future version in astropy/astropy#11608 but for now it's to make sure anyone whose looking closely will see this subtlety.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: docs/coordinates/solarsystem.rst [hunk 1]
```diff
@@ -129,6 +130,14 @@ For a list of the bodies for which positions can be calculated, do:
     to assume that ``get_body(time, 'sun')`` and ``get_sun(time)`` will give
     the same result.
 
+.. note :: 
+    When using JPL ephemerides, be aware that answers may change at levels that
+    can be surprising if you are not careful about understanding the frame you
+    are in or .  See for example the case of the DE440s ephemerides, which is
```
Comment (mhvk) [inline docs/coordinates/solarsystem.rst:136]: Weird end of sentence. Just delete "or"?

**Review by mhvk — COMMENTED** [review]
Yes, good idea! Just a problem with one sentence ending weirdly.

File: docs/coordinates/solarsystem.rst (hunk 1 above)
Comment (eteq) [reply docs/coordinates/solarsystem.rst:136]: Yep good point!  Will do:
```suggestion
    are in.  See for example the case of the DE440s ephemerides, which is
```

**Comment by eteq** [comment]
I'm guessing the CI is pretty jammed up, but at least locally the style checks and docs build succeeded. @mhvk, if you're happy with this you can approve and either merge or put merge-when-ci-passes?

**Review by mhvk — APPROVED** [review]

**Comment by saimn** [comment]
CI seems stuck, closing/reopening to trigger a new build.

**Comment by saimn** [comment]
Hmm Actions is not starting for this PR, not sure why ...

**Comment by pllim** [comment]
There is a `[ci skip]` directive in the second commit. You need to amend that commit and force push. 😬

**Comment by pllim** [comment]
Although the doc rendered fine, so I don't think we need the CI anyway. I am just going to merge. Thanks!

**Pull request merged**